src/main/java/com/ldtteam/structurize/storage/rendering/types/BoxPreviewData.java

```diff
@@ -10,6 +15,22 @@
  */
 public class BoxPreviewData
 {
+    public static final Codec<BoxPreviewData> CODEC = RecordCodecBuilder.create(
```

Comment:
Same here, making that a normal method + functional block seems much more readable

Reply:
this is easily readable? pos1 is blockpos type and getPos1 getter, same for others